spring-cloud-starter-stream-processor-twitter-sentiment/src/main/java/org/springframework/cloud/stream/app/twitter/sentiment/processor/TwitterSentimentTensorflowInputConverter.java

```diff
@@ -81,6 +81,10 @@ public TwitterSentimentTensorflowInputConverter(Resource vocabularLocation) {
 
 		try {
 
+			if (input instanceof byte[]) {
+				input = new String((byte[]) input);
+			}
+
 			if (input instanceof String) {
```

Comment:
Artem says why we need to convert it to String.

Reply:
+1